### PR TITLE
State we are using BCP 14

### PR DESCRIPTION
**Pull request by LPardue**
point 1 of #1783

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -123,7 +123,8 @@ substitute for RFC 7540 stream priority.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
-interpreted as described in {{!RFC2119}}.
+interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}} when, and only
+when, they appear in all captitals, as shown here.
```
Comment (martinthomson) [inline draft-ietf-httpbis-priority.md:127]:
```suggestion
```

File: draft-ietf-httpbis-priority.md [hunk 2]
```diff
@@ -123,7 +123,8 @@ substitute for RFC 7540 stream priority.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
```
Comment (martinthomson) [inline draft-ietf-httpbis-priority.md:125]:
```suggestion
{::boilerplate bcp14-tagged}
```

**Review by martinthomson — COMMENTED** [review]
There is another way.

**Review by kazuho — APPROVED** [review]
Nice! Thank you for the fix.

**Pull request merged**